### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by Cleecanth** [UWHealth/smph-department-sites#31]
There are plenty of places where this site's accessibility is a little weak:

* [x] Dropdown menus should have aria-roles ( [article on the topic](http://terrillthompson.com/blog/202) ) and the ability to be used via focus events.
* [x] Navigation should have role="navigation"
* [x] All anchors should have some kind of :focus styling

...more to be added as they come up. This will sort of be a catch-all issue.
